### ISSUE TITLE
Position/velocity from ephemeris

### ISSUE THREAD

**Issue by MichaelBonnet** [cassova/satellite-czml#4]
Is there a decently simple way to use this package in such a way that the satellite's position is populated using ephemeris rather than propagated from a TLE? For example, perhaps using STK attitude and ephemeris files.

**Comment by cassova** [comment]
I don't think so since I initially wrote this to work with just TLE data but I've been away from this for so long I just don't know.  Happy to review a PR to the code if you want to contribute.
